scripts/build-upload-a-docker-image.sh

```diff
@@ -107,6 +107,7 @@ else
 	    if [[ "$overwrite" == 'N' ]]; then
 	      check_overwrite "${IMAGE_TAGS[@]}"
 	    fi
+	    bash scripts/upload-docker-readme.sh "${component_name}" "${dir_arg}"/README.md
```

Comment:
+1 for having separate script!

I would prefer this to be called **after** the Docker image upload was successful. You can set a flag here and call the script after `docker buildx build` below